crates/iota-ledger-signer/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "iota-ledger-signer"
+version.workspace = true
+authors = ["IOTA Foundation <info@iota.org>"]
+edition = "2021"
+license = "Apache-2.0"
+publish = false
+
+[lints]
+workspace = true
+
+[dependencies]
+# external dependencies
+bcs.workspace = true
+bip32.workspace = true
+serde.workspace = true
```

Comment:
unused

Reply:
Fixed in https://github.com/iotaledger/iota/pull/7620/commits/a9063753da1379efb5bb7580dd78e4fd9ef4e2e5